mlflow/system_metrics/metrics/network_monitor.py

```diff
@@ -0,0 +1,21 @@
+"""Class for monitoring network stats."""
+
+
+import psutil
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+
+class NetworkMonitor(BaseMetricsMonitor):
+    def __init__(self, name="network"):
+        super().__init__(name)
+
+    def collect_metrics(self):
+        # Get network usage metrics.
+        network_usage = psutil.net_io_counters()._asdict()
```

Comment:
Is this global or restricted only to REPL processes?

Reply:
not 100% certain, but I think that's the global metrics: https://www.educative.io/answers/what-is-the-psutilnetiocounters-method